docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -16,12 +16,24 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 >**_NOTE:_** Currently only Minor Version Upgrades are support for Bare Metal clusters. No other aspects of the cluster upgrades are currently supported.
 >
 
+### Modular Cluster Upgrades
+
+To allow users to be explicit when deciding which clusters to upgrade, workload clusters are not affected by management cluster upgrades.
+Therefore, each cluster must be individually upgraded to enjoy the latest features on all clusters.
+
+When triggering a workload cluster upgrade after upgrading the management cluster, it will not only reflect the changes in the cluster spec, but also any changes derived from the difference in business logic between the old and new EKS-A cluster controllers.
+This difference in logic can trigger a machine rollout, even if the updates to the cluster spec didn't require one (for example, scaling down a worker node group).
+
 ### EKS Anywhere Version Upgrades
 
 We encourage that you stay up to date with the latest EKS Anywhere version, as new features, bug fixes, or security patches might be added in each release. You can find the content, such as supported OS versions and changelog, of each EKS Anywhere version on the [What’s New]({{< relref "../../../reference/changelog/" >}}) page.
 
 Download the [latest or target EKS Anywhere release](https://github.com/aws/eks-anywhere/releases/) and run `eksctl anywhere upgrade cluster` command to upgrade a cluster to a specific EKS Anywhere version.
 
+We provide a maximum skew version support of one EKS Anywhere minor version for management and workload clusters.
+This means the management cluster can only be one EKS Anywhere minor version after the workload clusters (such as v0.16 and v0.17).
+In the event that you want to upgrade your management cluster to a version that does not satisfy this condition, we suggest upgrading the workload cluster's EKS-A version first, followed by upgrading to your desired EKS-A version for the management cluster.
+
 **Skipping Amazon EKS Anywhere minor versions during cluster upgrade (such as going from v0.14 directly to v0.16) is NOT allowed.** EKS Anywhere team performs regular upgrade reliability testing for sequential version upgrade (e.g. going from version 0.14 to 0.15, then from version 0.15 to 0.16), but we do not perform testing on non-sequential upgrade path (e.g. going from version 0.14 directly to 0.16). You should not skip minor versions during cluster upgrade. However, you can choose to skip patch versions.
```

Comment:
This is not accurate. I want to soften this language. We aren't doing anything in the product to prevent a customer from skipping EKS-A minor versions when they upgrade. We should position it as "do it at your own risk"